src/main/java/org/mvel2/asm/Opcodes.java

```diff
@@ -58,6 +58,7 @@
     int V1_6 = 0 << 16 | 50;
     int V1_7 = 0 << 16 | 51;
     int V1_8 = 0 << 16 | 52;
+    int V1_9 = 0 << 16 | 53;
```

Comment:
Actually looking at the code again, we really should upgrade the ASM as well. The tests are passing now, but I have no idea if the ASM version currently in use is JDK 9 compatible.